src/lib/components/index/github-card.svelte

```diff
@@ -29,7 +29,7 @@
 
 <style lang="postcss">
   .card:hover::before {
-    @apply absolute -z-10 h-[500px] w-[100px] bg-[#000000];
+    @apply absolute -z-10 h-[500px] w-[100px] bg-[#fff];
```

Comment:
This will work only on dark mode. If it is on light mode it will be not visible.
This suggestion will fix it:
```suggestion
    @apply absolute -z-10 h-[500px] w-[100px] bg-black dark:bg-white;
```